Mathlib/MeasureTheory/Integral/SetIntegral.lean

```diff
@@ -1409,7 +1409,7 @@ section ParametricIntegral
 
 variable [NormedAddCommGroup E]
 
-variable {α β F G 𝕜 : Type*} [TopologicalSpace α] [TopologicalSpace β] [MeasurableSpace β]
+variable {X β F G 𝕜 : Type*} [TopologicalSpace X] [TopologicalSpace β] [MeasurableSpace β]
```

Comment:
Why change α but not β? (Might be better in a separate PR anyway)

Reply:
Because `X` occurs in the following lemma :-)
I've renamed all `α` in this file, filed as #11131.